Misc/NEWS.d/next/Library/2020-03-05-00-57-49.bpo-39828.yWq9NJ.rst

```diff
@@ -0,0 +1 @@
+Fix :mod:`json.tool` to ignore :exc:`BrokenPipeError`. Patch by Dong-hee Na.
```

Comment:
The error is not exactly "ignored", but "handled" or "catched". You should rephrase the NEWS entry.